bin/node/cli/tests/temp_base_path_works.rs

```diff
@@ -32,6 +32,7 @@ use std::{
 
 pub mod common;
 
+#[ignore]
```

Comment:
Do we still want to ignore this test?

Reply:
Was probably randomly failing https://github.com/paritytech/substrate/pull/13505